src/index.ts

```diff
@@ -34,21 +34,28 @@ export default {
     const res = await fetch(`${Config.STATUS_URL}/api/v2/incidents.json`)
     const json = await res.json<IncidentResponse>();
 
-    await Promise.all(json.incidents.map(async incident => {
-      const kv = await env.KV.get<Incident>(incident.id, 'json');
+    // Track if this is the first run, so we can ignore resolved incidents
+    const firstRun = await env.KV.get('firstRun', 'text') === null;
+    if (firstRun) await env.KV.put('firstRun', new Date().toISOString());
 
+    await Promise.all(json.incidents.map(async incident => {
+      const kv = await env.KV.get<StoredIncident>(incident.id, 'json');
       console.log('-----\nIncident ' + incident.id + ' in KV: ' + (kv !== null) + '\n-----');
 
+      // On the first run, ignore any incidents that are already resolved
+      // We'll store them as skipped so we don't keep checking them
+      if (firstRun && incident.status === 'resolved') {
+        await env.KV.put(incident.id, JSON.stringify({ skipped: true }));
```

Comment:
Wonder if we should just set a null value and use metadata instead. Not super opinionated on this though

Reply:
With #13 lurking I figured sticking to actual stored data was better?